### PR TITLE
CON-3303 Migrate data locks review to V2

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingDataLockRequestChangesTests.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        [Test]
+        public async Task VerifyRejectIsCalled_WhenNotApprovingRequest()
+        {
+            _viewModel.AcceptChanges = false;
+            await _fixture.DataLockRequestChanges(_viewModel);
+            _fixture.VerifyRejectIsCalledOnApi();
+        }
+
+        [Test]
+        public async Task VerifyRejectIsCalled_WhenApprovingRequest()
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingDataLockRequestChangesTests.cs:35]: Should this say `VerifyAcceptIsCalled`?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/DataLockRequestChangesRequestToViewModelMapper.cs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+using SFA.DAS.CommitmentsV2.Api.Types.Responses;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.Edit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers.Apprentice
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/DataLockRequestChangesRequestToViewModelMapper.cs:11]: should we have tests for this ?

**Review by cofaulco — COMMENTED** [review]
1 comment re. unit tests

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingDataLockRequestChangesTests.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Controllers/ApprenticeControllerTests/WhenPostingDataLockRequestChangesTests.cs]: Done

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/DataLockRequestChangesRequestToViewModelMapper.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/DataLockRequestChangesRequestToViewModelMapper.cs:11]: Added some tests for model mapper

**Review by cofaulco — APPROVED** [review]

**Pull request merged**